src/main/java/frc/robot/auto/Path.java

```diff
@@ -34,27 +34,27 @@ public Path(ArrayList<PathPoint> Points) {
             PathPoint lastPoint = points.get(i-1);
             // Calculate distance from last point
             point.defineDistanceAlongPath( // Set to Z
-                point.getPosMeters().getDistance(
-                    lastPoint.getPosMeters()
+                point.posMeters.getDistance(
+                    lastPoint.posMeters
                     // Add last value
-                ) + lastPoint.getDistanceAlongPathMeters()
+                ) + lastPoint.distanceAlongPath
             );
         }
 
         // Correct or implement orientation for each point.
         // Handle first point
-        if (firstPoint.getOrientation() == null) {
-            firstPoint.setOrientation(new Rotation2d());
+        if (firstPoint.orientation == null) {
+            firstPoint.orientation = new Rotation2d();
```

Comment:
Will this cause the robot to always face 0 degrees on any path that doesn't have rotations?